dropwizard-metrics/4.0/pom.xml

```diff
@@ -0,0 +1,35 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.wavefront</groupId>
+    <artifactId>wavefront</artifactId>
+    <version>3.7-SNAPSHOT</version>
+    <relativePath>../../pom.xml</relativePath>
+  </parent>
+  
+  <artifactId>metrics-wavefront</artifactId>
```

Comment:
OK, I'll switch to that for artifact idea and change the version to 3.7-SNAPSHOT to match everything else in the project. This way it'll still remain clear that this version is for use with DropWizard Metrics 4.0 while still allowing us to have an independent version number for the Wavefront Reporter that we can increment as we add features and fix bugs.